jpegenc/models/rgb2ycbcr/commons.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/python
```

Comment:
i will remove common.py since it isnt used anymore in my code. also the roundind occurs inside the code in rgb2ycbcr.py